GitExtUtils/GitDubiousOwnershipSecurityException.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace GitExtUtils;
+
+public class GitDubiousOwnershipSecurityException : Exception
+{
+    private const string _dubiousOwnershipSecurityConfigString = "config --global --add safe.directory";
+    public const string TrustAllReposCommand = "config --global --add safe.directory *";
+    public string Error { get; }
+    public string Command { get; }
+
+    public static bool IsGitDubiousOwnershipSecurityError(string error)
```

Comment:
This doesn't feel right to have it here. Coupled with the comment above, this check should be at the UI layer where we made a decision about what UI to present to the user.